restructuring AIT3 docs + small cli bugs

wintertoro opened the pull request:
I want to restructure. the AIT3 docs in this way as I think it would be significantly clearer. I also fixed some small bugs like max-gas missing from CLI instructions. 

some links might be broken, but I can fix that after this goes through

gregnazario commented on developer-docs-site/docs/nodes/ait/connect-to-testnet.md:
```diff
@@ -153,7 +153,8 @@ At this point you already used your owner account to initialized a validator sta
       --pool-address <owner-address> \
-      --profile ait3-operator
+      --profile ait3-operator \ 
+      --max-gas 10000 # You can play with the amt of max gas here. Make sure you have sent your operator enough tokens for them to pay gas fee
```
`You can adjust the amount of max gas here. Please make sure you have sent your operator enough tokens to pay for the gas fee.`

gregnazario commented on developer-docs-site/docs/nodes/ait/additional-doc.md:
```diff
@@ -114,7 +114,8 @@ We now have a UI to support some staking operation, but in any case if you need
     --amount <amount> \
-    --profile ait3-owner
+    --profile ait3-owner \
+    --max gas 5000 # you can play around with the max gas here
```
We should be consistent on the comment

gregnazario commented on developer-docs-site/docs/nodes/ait/connect-to-testnet.md:
```diff
@@ -227,13 +193,64 @@ You can check the details about node liveness definition [here](https://aptos.de
 ## Leaving Validator Set
 
-A node can choose to leave validator set at anytime, or it would happen automatically when there's not sufficient stake on the validator account. To leave validator set, you can perform the following steps:
+A node can choose to leave validator set at anytime, or it would happen automatically when there's not sufficient stake on the validator account. Before you shutdown the node, you should also need to make sure you leave validator set first (will take effect in next epoch). To leave validator set, you can perform the following steps:
```
`Before you shutdown the node, you also need to leave the validator set first.`

gregnazario commented on developer-docs-site/docs/nodes/ait/connect-to-testnet.md:
````diff
@@ -8,40 +8,6 @@ sidebar_position: 14
-
-  ```
-  aptos init --profile ait3-owner \
-    --rest-url https://ait3.aptosdev.com
````
Why is this gone now?

wintertoro replied:
its a duplicate

gregnazario commented on developer-docs-site/docs/nodes/ait/connect-to-testnet.md:
```diff
@@ -227,13 +193,64 @@ You can check the details about node liveness definition [here](https://aptos.de
+### Using source code
+
+- Stop your node.
+- Remove the data directory: `rm -rf <your-data-directory>`
```
We should avoid telling people to use `rm -rf` forcing, is not preferred because you can really do some damage.

wintertoro replied:
what is the suggested language here?

gregnazario commented on developer-docs-site/docs/nodes/ait/connect-to-testnet.md:
````diff
@@ -227,13 +193,64 @@ You can check the details about node liveness definition [here](https://aptos.de
     ```
+## Shutdown Nodes for Incentivized Testnet
+
+If you have ensured that your node left the validator set, you can now take down the validator node and cleanup the resources used by the node.
````
I'm not sure we have to tell people to cleanup all of these resources

wintertoro replied:
This was originally there

gregnazario reviewed (commented):
Everything is getting moved around all over the place.

What have you added that isn't just moving docs?

wintertoro commented:
I cleaned it up, there’s a ton of replication between the docs. 

Like initialize staking is spread out over three docs originally. So I put all the stuff to do with CLI, voting, and staking (incl) Petra wallet, in one doc. Then all the stuff to do with joining the network is consolidated in another doc. I hate having to jump between three docs to execute one step. 

The actual bug change is just the max gas thing we talked about. 

I also removed duplicates of the CLI example code.


> On Aug 31, 2022, at 14:25, Greg Nazario ***@***.***> wrote:
> 
> 
> @gregnazario commented on this pull request.
> 
> Everything is getting moved around all over the place.
> 
> What have you added that isn't just moving docs?
> 
> In developer-docs-site/docs/nodes/ait/connect-to-testnet.md <https://github.com/aptos-labs/aptos-core/pull/3704#discussion_r960032289>:
> 
> > @@ -153,7 +153,8 @@ At this point you already used your owner account to initialized a validator sta
>      ```
>      aptos node join-validator-set \
>        --pool-address <owner-address> \
> -      --profile ait3-operator
> +      --profile ait3-operator \ 
> +      --max-gas 10000 # You can play with the amt of max gas here. Make sure you have sent your operator enough tokens for them to pay gas fee
> You can adjust the amount of max gas here. Please make sure you have sent your operator enough tokens to pay for the gas fee.
> 
> In developer-docs-site/docs/nodes/ait/additional-doc.md <https://github.com/aptos-labs/aptos-core/pull/3704#discussion_r960032495>:
> 
> > @@ -114,7 +114,8 @@ We now have a UI to support some staking operation, but in any case if you need
>    ```
>    aptos stake add-stake \
>      --amount <amount> \
> -    --profile ait3-owner
> +    --profile ait3-owner \
> +    --max gas 5000 # you can play around with the max gas here
> We should be consistent on the comment
> 
> In developer-docs-site/docs/nodes/ait/connect-to-testnet.md <https://github.com/aptos-labs/aptos-core/pull/3704#discussion_r960033111>:
> 
> >  
>  ## Leaving Validator Set
>  
> -A node can choose to leave validator set at anytime, or it would happen automatically when there's not sufficient stake on the validator account. To leave validator set, you can perform the following steps:
> +A node can choose to leave validator set at anytime, or it would happen automatically when there's not sufficient stake on the validator account. Before you shutdown the node, you should also need to make sure you leave validator set first (will take effect in next epoch). To leave validator set, you can perform the following steps:
> Before you shutdown the node, you also need to leave the validator set first.
> 
> In developer-docs-site/docs/nodes/ait/connect-to-testnet.md <https://github.com/aptos-labs/aptos-core/pull/3704#discussion_r960033639>:
> 
> > -## Initializing staking pool
> -
> -In AIT3 we will have UI support to allow owner managing the staking pool, see details [here](https://aptos.dev/nodes/ait/steps-in-ait3#initialize-staking-pool), if you've already done this through the UI, you can igore this step and jump into "Bootstrapping validator node". 
> -
> -Alternatively, you can also use CLI to intialize staking pool:
> -
> -- Initialize CLI with your wallet **private key**, you can get in from Settings -> Credentials
> -
> -  ```
> -  aptos init --profile ait3-owner \
> -    --rest-url https://ait3.aptosdev.com
> Why is this gone now?
> 
> In developer-docs-site/docs/nodes/ait/connect-to-testnet.md <https://github.com/aptos-labs/aptos-core/pull/3704#discussion_r960034001>:
> 
> >  
>  1. Leave validator set (will take effect in next epoch)
>  
>      ```
>      aptos node leave-validator-set --profile ait3-operator --pool-address <owner-address>
>      ```
> +## Shutdown Nodes for Incentivized Testnet
> +
> +If you have ensured that your node left the validator set, you can now take down the validator node and cleanup the resources used by the node.
> +
> +### Using source code
> +
> +- Stop your node.
> +- Remove the data directory: `rm -rf <your-data-directory>`
> We should avoid telling people to use rm -rf forcing, is not preferred because you can really do some damage.
> 
> In developer-docs-site/docs/nodes/ait/connect-to-testnet.md <https://github.com/aptos-labs/aptos-core/pull/3704#discussion_r960037057>:
> 
> >  
>  1. Leave validator set (will take effect in next epoch)
>  
>      ```
>      aptos node leave-validator-set --profile ait3-operator --pool-address <owner-address>
>      ```
> +## Shutdown Nodes for Incentivized Testnet
> +
> +If you have ensured that your node left the validator set, you can now take down the validator node and cleanup the resources used by the node.
> I'm not sure we have to tell people to cleanup all of these resources
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/aptos-labs/aptos-core/pull/3704#pullrequestreview-1092521719>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMTAXMTT3WLJPCVIIUDTTBLV37EUFANCNFSM6AAAAAAQBXYHOE>.
> You are receiving this because you authored the thread.
>
